meson.build

```diff
@@ -58,7 +57,7 @@ if cpp.get_argument_syntax() == 'gcc' and cpp.version().version_compare('<9')
   endif
 endif
 
-brotli_dep = dependency('libbrotlidec', disabler: true, required: false)
+brotli_dep = dependency('libbrotlidec', disabler: true, required: get_option('bmff'))
```

Comment:
Maybe it doesn't matter, but this is not how CMake currently works - [Brotli is an orthogonal option, BMFF can still be enabled without it,](https://github.com/Exiv2/exiv2/blob/93f5acbbff7dacc66cfa02a56957c5746a7c107c/cmake/findDependencies.cmake#L49-L51) so perhaps we want to mirror this behaviour for Meson?